snuba/utils/logging.py

```diff
@@ -0,0 +1,27 @@
+# Python log levels are:
+# CRITICAL 50
+# ERROR    40
+# WARNING  30
+# INFO     20
+# DEBUG    10
+# NOTSET    0
+# syslog(3) levels are:
+# EMERGENCY 0
+# ALERT     1
+# CRITICAL  2
+# ERROR     3
+# WARNING   4
+# NOTICE    5
+# INFO      6
+# DEBUG     7
+
+
+def pylog_to_syslog_level(level: int) -> int:
+    if level == 0:  # NOTSET
+        return 7  # DEBUG, to process all messages
+
+    syslog_level = 8 - level // 10
+    if syslog_level <= 5:
+        syslog_level -= 1  # Adjust for lack of NOTICE level on Python
+
+    return syslog_level
```

Comment:
Yes math works but a mapping might be easier to reason about and maintain.

```python
def pylog_to_syslog_level(level: int) -> int:
    return {   # Python    syslog
        50: 0, # CRITICAL  EMERGENCY
        40: 1, # ERROR     ALERT
        # ...
    }.get(level, 3)
```

Reply:
eh, I think you are right, will refactor this.